client/src/lib/routes.js

```diff
@@ -317,6 +325,29 @@ product_id   int         ID for the product being added to the cart
 
 Status: 201 CREATED
 */
+export const addToCart = ((sessionId, productId, callback = arrowFunc) => {
+  const path = `${apiPath}/cart/`;
+  fetch(path, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    // body: JSON.stringify({ user_session: int, product_id: int }),
+    body: JSON.stringify({
+      user_session: sessionId,
+      product_id: productId,
+      active: 1,
+    }),
+  })
+    .then((data) => {
+      // console.log('data in fetch post request: ', data);
+      callback(null, data);
+    })
+    .catch((error) => {
+      // console.log('error in fetch post request: ', error);
+      callback(error, null);
+    });
```

Comment:
Fetch looks super cool, but if you're curious I think this could be an axios request in about half of the lines it is now. Totally optional tho, looks great.

Reply:
It definitely _could_ but I am going to stick with fetch for now.